apis/user-java/helm/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: 'api-user-svc'
```

Comment:
Let's use `api-user-profile-svc` since this service name conflicts: https://github.com/Azure-Samples/openhack-devops-team/blob/28733e259832b4e709df4e2b20b538d60d37622f/apis/userprofile/helm/templates/service.yaml#L4